qemu/tests/blockdev_inc_backup_xpt_nonexist_bitmap.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import string
+
+from provider.blockdev_base import BlockdevBaseTest
+from provider.nbd_image_export import QemuNBDExportImage
+
+from virttest import utils_misc
+
+from avocado.utils import process
```

Comment:
Better to move these after standard modules:
standard
avocado
vt
tp-qemu

Reply:
Have updated, thanks zhenchao.